File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Icon.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Icon{

	@SerializedName("thumbnails")
	private List<ThumbnailsItem> thumbnails;

	public void setThumbnails(List<ThumbnailsItem> thumbnails){
		this.thumbnails = thumbnails;
	}

	public List<ThumbnailsItem> getThumbnails(){
		return thumbnails;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Icon.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Icon : Parcelable {
    @SerializedName("iconType")
    var iconType: String? = null
    override fun toString(): String {
        return "Icon{" +
                "iconType = '" + iconType + '\'' +
                "}"
    }
}